core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -115,7 +115,7 @@ public boolean getRequiresConfirmation() {
      * @return permission required for user to access this management link, in addition to {@link Jenkins#ADMINISTER}
      */
     public @CheckForNull Permission getRequiredPermission() {
-        return null;
+        return Jenkins.ADMINISTER;
```

Comment:
This does hugely impact system read though, and it doesn't seem to have been considered in https://github.com/jenkinsci/jep/pull/269/files at all

The manage page is now completely empty if I branch JEP-223 and JEP-224 and merge them together.

Options I've looked at so far,

1. revert this change and go back to null, system read continues to work, manage will see everything
2. change it to system read, then you get `manager is missing the Overall/SystemRead permission`

Reply:
I addressed @timja's concerns in https://github.com/jenkinsci/jenkins/pull/4501#issuecomment-586597011

Reply:
I agree that it is ok as an intermediate solution. `permission required for user to access this management link, in addition to {@link Jenkins#ADMINISTER}` already implies that there will be rework needed to make it compatible with JEP-224. Since the JEP-223 is about landing as experimental, we can afford further breaking changes in the implementation if needed for JEP-224